cfme/services/myservice/ui.py

```diff
@@ -370,6 +371,10 @@ def step(self, *args, **kwargs):
 
     def resetter(self, *args, **kwargs):
         self.prerequisite_view.navigation.select('Services', 'My Services')
+        if self.obj.rest_api_entity.retired:
```

Comment:
I am not sure is this good solution. The service can be here or there depending on its Retirement state. So I am changing the path depnding on what state ws read from rest_api.

Reply:
@jaryn this navigation registered like `All`. lets put this pointing to `Active Services` in `resetter` 

if you would like to point `retired` let lets add one more navigation destination against `MyService` which will point `Retired Services` like

```
@navigator.register(MyService, 'AllRetired')
class MyServiceDetails(CFMENavigateStep):
    VIEW = MyServiceDetailView

    prerequisite = NavigateToSibling('All')

    def step(self, *args, **kwargs):
        self.prerequisite_view.myservice.tree.click_path('Retired Services')
```

Reply:
I moved the logic to the "Details". I am not convinced that creating some property is any good.